src/jwplayer/players/DesktopArticleVideoPlayer/DesktopArticleVideoPlayer.tsx

```diff
@@ -98,7 +98,7 @@ const DesktopArticleVideoPlayer: React.FC<DesktopArticleVideoPlayerProps> = ({ v
 						<JwPlayerWrapper
 							config={getArticleVideoConfig(videoDetails)}
 							onReady={(playerInstance) => articlePlayerOnReady(videoDetails, playerInstance)}
-							stopAutoAdvanceOnExitViewport={true}
+							stopAutoAdvanceOnExitViewport={false}
```

Comment:
Unless you want to keep this line for where to re-enable it, you can just remove the prop and it'll default to `false`.

Reply:
That's a good point. I'll probably keep it as a reminder. We'll be turning this back on at some point.